.github/workflows/integration.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Run your tests with reporting
         if: ${{ contains(github.workflow_ref, 'nightly') }}
         working-directory: tests/acceptance
-        run: npx pwc --project-id ${{ secrets.CURRENTS_PROJECT_ID }} --key ${{ secrets.CURRENTS_RECORD_KEY }} -- --project ${{ matrix.name }} --trace=on
+        run: npx pwc --project-id ${{ secrets.CURRENTS_PROJECT_ID }} --key ${{ secrets.CURRENTS_RECORD_KEY }} -- --project "${{ matrix.major && format('%s-major', matrix.name) || matrix.name }}" --trace=on
```

Comment:
AFAIK, project is the playwright project, I think there's no major project.

Reply:
We could add one that actually checks if it's in "major" mode

Reply:
Oh I see, I assumed this was just used to identify the testsuite for currents.dev, I'll revisit this

Reply:
I'm now setting a unique [Build ID](https://docs.currents.dev/guides/ci-build-id#what-is-ci-build-id), resulting in [separate runs](https://github.com/shopware/shopware/pull/5895#issuecomment-2551033283).